### ISSUE TITLE
missing ip core repositories

### ISSUE THREAD

**Issue by qermit** [wzab/vextproj#1]

**Comment by wzab** [comment]
The problem seems to be associated with attempt to run it on Windows.
I'm sorry, but Vextproj is Linux oriented. I do not warrant that my demos will run on Windows machines.

**Comment by wzab** [comment]
OK. The problem is explained. The user has tried to change the required Vivado version in the proj_def.tcl, to compile the design with an earlier version of Vivado.
Of course it does not change the saved BD file, which was prepared in 2016.1.
That's the reason why IP was not found...

**Comment by wzab** [comment]
I have checked VEXTPROJ on a Windows machine with Vivado 2016.1
I've started Vivado.
In the Tcl console I've changed directory to the location where I've downloaded the VEXTPROJ, than I changed directory to version_2/demo_wb/hdl
Next I have started the command "source eprj_create.tcl"
After it was successfully finished, I have closed the project, and next executed the command "source eprj_build.tcl".
Everything worked correctly and the bitstream was created.
Probably it should be possible to prepare a batchfile similar to the Linux "build.sh", but i couldn't find the equivalent for "settings64.sh" which would allow me to prepare environment (e.g., PATH) for that.
Anyway it seems that at least this demo works.
Probably there may be problems with using VCS commands and/or exec command, as they rely on Linux-like OS.
